src/black_primer/lib.py

```diff
@@ -176,6 +180,30 @@ async def git_checkout_or_rebase(
     return repo_path
 
 
+def handle_PermissionError(
+    func: Callable, path: Path, exc: Tuple[Any, Any, Any]
+) -> None:
+    """
+    Handle PermissionError during shutil.rmtree.
+
+    This checks if the erroring function is either 'os.rmdir' or 'os.unlink', and that
+    the error was EACCES (i.e. Permission denied). If true, the path is set writable,
+    readable, and executable by everyone. Finally, it tries the error causing delete
+    operation again.
+
+    If the check is false, then the original error will be reraised as this function
+    can't handle it.
+    """
+    excvalue = exc[1]
+    LOG.debug(f"Handling {excvalue} from {func.__name__} ... ")
```

Comment:
@hugovk probably would recommend this :)
> It's a minor change for sure, and achieves consistency with the rest of the codebase, and uses the most common form.
```suggestion
    LOG.debug(f"Handling {excvalue} from {func.__name__}... ")
```

Reply:
I really don't think debug output and white space is a super impactful thing to care about ...